candump.c

```diff
@@ -236,6 +281,16 @@ int main(int argc, char **argv)
 	struct timeval tv, last_tv;
 	struct timeval timeout, timeout_config = { 0, 0 }, *timeout_current = NULL;
 	FILE *logfile = NULL;
+	struct netlink_struc {
+		int s;
+		struct sockaddr_nl saddr;
+	} netlink_s = {
+			.s = socket(AF_NETLINK, SOCK_RAW, NETLINK_ROUTE),
```

Comment:
Sorry for not beeing clear in the first review, better use only C99 initializers for constant types. So move the: `s = socket();` next to the `bind()`.